ksql-functional-tests/src/test/java/io/confluent/ksql/test/PlannedTestGeneratorTest.java

```diff
@@ -15,24 +15,14 @@
 
 package io.confluent.ksql.test;
 
-import org.apache.kafka.test.IntegrationTest;
-import org.junit.ClassRule;
+import io.confluent.ksql.test.planned.PlannedTestGenerator;
+import org.junit.Ignore;
 import org.junit.Test;
-import org.junit.experimental.categories.Category;
-import org.junit.rules.TemporaryFolder;
 
-/**
- * Do not combine this with `TopologyFileGenerator` as mvn will ignore the tests as the class
- * does not end in `Test`.
- */
-@Category(IntegrationTest.class)
-public final class TopologyFileGeneratorTest {
-
-    @ClassRule
-    public static final TemporaryFolder TMP = new TemporaryFolder();
-
-    @Test
-    public void shouldGenerateTopologies() throws Exception {
-        TopologyFileGenerator.generateTopologies(TMP.newFolder().toPath());
-    }
+public class PlannedTestGeneratorTest {
+  @Test
+  @Ignore
```

Comment:
Why is this test being ignored?

Reply:
It's more a tool for generating new plans than an actual test. You can just run the "test" in your IDE and have any new plans generated for you.